src/main.c

```diff
@@ -222,7 +230,16 @@ int main(int argc, char *argv[])
 
     if (invert_reset)
     {
-        mode |= MODE_INVERT_RESET;
+        if (mode == 4)
+        {
+            mode = 4;
+            printf("Using mode 5. INVERT RESET command will not be applied.\r\n");
```

Comment:
Venting, not expecting you to change:  I hate that they used 0-base counting for the variable, but 1-based counting for the human readable mode number.  That is incredibly unintuitive.  I'd have rather they either use an Enum, or just not use `mode == 0`.  *sigh*  Just venting.  Don't change the rest of their code.

Reply:
Oh, dang.  See below in rl78.c, line 61.  They use #defines for modes.

Look for `#define MODE_UART_1` and see if you can add `#define MODE_UART_5 (4)`, and use `if (MODE_UART_1 == (mode&MODE_UART))` instead of `if (mode == 4)`